### ISSUE TITLE
Error: Invalid value at 'track_config.releases[0].status' (type.googleapis.com/google.play.publishingapi.v3.Release.Status), ""

### ISSUE THREAD

**Issue by Bubelbub** [r0adkll/upload-google-play#85]
**Describe the bug**
When specify a track then I get a error message.

**Workflow Step Configuration**

```yml
- uses: r0adkll/upload-google-play@v1
  if: ${{ steps.regex-match.outputs.match != '' }}
  with:
    serviceAccountJsonPlainText: ${{ secrets.SERVICE_ACCOUNT_JSON }}
    packageName: org.example
    releaseFiles: ${{steps.sign.outputs.signedReleaseFile}}
    track: production
```

**Step Debugging**

- [X] I have enabled [Step Debug Logging](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/enabling-debug-logging#enabling-step-debug-logging)

```
2021-10-29T11:56:06.0427623Z ##[group]Run r0adkll/upload-google-play@v1
2021-10-29T11:56:06.0428181Z with:
2021-10-29T11:56:06.0445140Z   serviceAccountJsonPlainText: ***
2021-10-29T11:56:06.0445863Z   packageName: org.example
2021-10-29T11:56:06.0446636Z   releaseFiles: android/app/build/outputs/bundle/release/app-release.aab
2021-10-29T11:56:06.0447322Z   track: production
2021-10-29T11:56:06.0447854Z   inAppUpdatePriority: 0
2021-10-29T11:56:06.0448349Z env:
2021-10-29T11:56:06.0448908Z   JAVA_HOME: /opt/hostedtoolcache/Java_Adopt_jdk/11.0.12-7/x64
2021-10-29T11:56:06.0449525Z   ANDROID_HOME: /root/.android/sdk
2021-10-29T11:56:06.0450069Z   ANDROID_SDK_ROOT: /root/.android/sdk
2021-10-29T11:56:06.0450780Z   SIGNED_RELEASE_FILE: android/app/build/outputs/bundle/release/app-release.aab
2021-10-29T11:56:06.0451441Z ##[endgroup]
2021-10-29T11:56:06.3575007Z Creating a new Edit for this release
2021-10-29T11:56:07.0792806Z Validating track 'production'
2021-10-29T11:56:07.3162949Z Uploading android/app/build/outputs/bundle/release/app-release.aab
2021-10-29T11:56:56.7899882Z Successfully uploaded 1 artifacts
2021-10-29T11:56:56.7902604Z Adding 1 artifacts to release on 'production' track
2021-10-29T11:56:57.3780046Z ##[error]Invalid value at 'track_config.releases[0].status' (type.googleapis.com/google.play.publishingapi.v3.Release.Status), ""
```

**Comment by yairm210** [comment]
Looks like a copy of #80 , try setting the version manually as discussed there, worked for me

**Comment by storeyleon** [comment]
> Looks like a copy of #80 , try setting the version manually as discussed there, worked for me

Yep, works fine with `r0adkll/upload-google-play@v1.0.15`

**Comment by danielehrhardt** [comment]
> > Looks like a copy of #80 , try setting the version manually as discussed there, worked for me
> 
> Yep, works fine with `r0adkll/upload-google-play@v1.0.15`

```
Run r0adkll/upload-google-play@v1.0.16
Creating a new Edit for this release
Validating track 'production'
Uploading android/app/build/outputs/bundle/release/app-release.aab
Successfully uploaded 1 artifacts
Adding 1 artifacts to release on 'production' track
Error: Invalid value at 'track_config.releases[0].status' (type.googleapis.com/google.play.publishingapi.v3.Release.Status), ""
```

**Comment by rchampa** [comment]
I'm facing the same issue wit the following setup.
`
- name: Deploy to Play Store (Alpha)
        id: deploy
        uses: r0adkll/upload-google-play@v1.0.16
        with:
          serviceAccountJsonPlainText: ${{ secrets.SERVICE_ACCOUNT_JSON }}
          packageName: es.algoritmo.githubactions
          releaseFiles: ${{steps.sign_app.outputs.signedReleaseFile}}
          track: alpha
          whatsNewDirectory: whatsnew/
`

**Comment by yairm210** [comment]
You both are using `1.0.16`, which is the broken version, and not `1.0.15`, which is the non-broken version.

**Comment by patrickbussmann** [comment]
@yairm210 why then there is no "new" release with a working version?
I dont see `1.0.17` 😥

**Comment by yairm210** [comment]
I don't know, I'm not a maintainer ¯\_(ツ)_/¯
Just someone who encountered the same problem and is trying to help :)

**Comment by kelvinmarshall** [comment]
Adding the status property fixes the issue.

**Comment by boswelja** [comment]
This is fixed by #87, and will be available in the next release

**Comment by patrickbussmann** [comment]
@boswelja I had this issue now again.
Tried with `@master` and with `@v1.0.15`.
And the merge #127 is already merged.

So I think there is another issue?
